tests/store/tracking/test_file_store.py

```diff
@@ -1227,6 +1248,39 @@ def test_search_runs_run_id(self):
         )
         assert result == []
 
+    def test_search_runs_start_time_alias(self):
+        fs = FileStore(self.test_root)
+        exp_id = fs.create_experiment("test_search_runs_start_time_alias")
+        # Set start_time to ensure the search result is deterministic
+        run1 = fs.create_run(exp_id, user_id="user", start_time=1, tags=[], run_name="name")
+        run2 = fs.create_run(exp_id, user_id="user", start_time=2, tags=[], run_name="name")
+        run_id1 = run1.info.run_id
+        run_id2 = run2.info.run_id
+
+        result = fs.search_runs(
+            [exp_id],
+            filter_string="attributes.run_name = 'name'",
+            run_view_type=ViewType.ACTIVE_ONLY,
+            order_by=["attributes.start_time DESC"],
+        )
+        assert [r.info.run_id for r in result] == [run_id2, run_id1]
+
+        result = fs.search_runs(
+            [exp_id],
+            filter_string="attributes.run_name = 'name'",
+            run_view_type=ViewType.ACTIVE_ONLY,
+            order_by=["attributes.created ASC"],
+        )
+        assert [r.info.run_id for r in result] == [run_id1, run_id2]
+
+        result = fs.search_runs(
+            [exp_id],
+            filter_string="attributes.run_name = 'name'",
+            run_view_type=ViewType.ACTIVE_ONLY,
+            order_by=["attributes.Created DESC"],
+        )
```

Comment:
all of these filter strings appear to be the same. Can we make them different? Also, can we add tests for "Created" in the filter?

Reply:
https://github.com/mlflow/mlflow/pull/7492#discussion_r1044896778

Reply:
Then the title of the test case seems to be misleading

Reply:
I see - can we test filtering on Created too?